### DISCUSSION TITLE
How to display in GitHub the Pluto.jl notebook?

### DISCUSSION THREAD

**Discussion by a9lara** [fonsp/Pluto.jl#769] [Questions about using Pluto] [answered]
If I upload a JUpyter notebook, it can be seen displayed on GitHub. If I upload a Pluto notebook, it does not display on GitHub, it is only plain text.

Is there any way Pluto Notebooks cane be displayed on GitHub repositories?

Regards 

Armando

**Comment by fonsp** [comment, marked as answer]
Hi Armando,

We made the choice to not include cell outputs in the notebook file (`.jl`). This has some advantages, but the disadvantage is that just the notebook file is not enough to show a preview. For github to show previews, they would need to run every Pluto notebook that we put online.

The current export options with output are HTML and PDF. HTML does not render inside github, but PDF does! So my suggestion is to export your notebooks to PDF, and put those in the repository.

-fons

**Reply by a9lara** [reply]
Thank you very much. :)
